covid_xprize/examples/prescriptors/neat/utils.py

```diff
@@ -4,17 +4,29 @@
 import subprocess
 import urllib.request
 import pandas as pd
+from pathlib import Path
+
+from tempfile import NamedTemporaryFile
 
 from covid_xprize.validation.scenario_generator import get_raw_data, generate_scenario
 
+# URL for Oxford data
 DATA_URL = "https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv"
-ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
-DATA_PATH = os.path.join(ROOT_DIR, 'data')
-HIST_DATA_FILE_PATH = os.path.join(DATA_PATH, 'OxCGRT_latest.csv')
 
-PREDICT_MODULE = 'covid_xprize/standard_predictor/predict.py'
-TMP_PRED_FILE_NAME = 'tmp_predictions_for_prescriptions/preds.csv'
-TMP_PRESCRIPTION_FILE = 'tmp_prescription.csv'
+# Path to where this script lives
+ROOT_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
+
+# Data directory (we will download the Oxford data to here)
+DATA_PATH = ROOT_DIR / 'data'
+
+# Path to Oxford data file
+HIST_DATA_FILE_PATH = DATA_PATH / 'OxCGRT_latest.csv'
+
+# Path to predictor module
+PREDICT_MODULE = ROOT_DIR.parent.parent.parent / 'standard_predictor' / 'predict.py'
+
+TMP_PRED_FILE_NAME = NamedTemporaryFile().name
```

Comment:
These don't seem to be needed after the script is done, so I made them true "temp" files. They will be created in the system temporary directory (`/tmp` on UNIX-y systems) and auto deleted when closed.